src/scenes/technical/index.js

```diff
@@ -23,18 +42,35 @@ export default class Technical extends Component {
             <Divider hidden />
             <Header size='big'>Choose a Category</Header>
             <Divider hidden />
-            <Grid columns={2} stackable relaxed>
+            <Grid columns={3} stackable relaxed>
               <Grid.Row>
                 <Grid.Column>
 
-                  <Link to='/technical/knowledge-base'>
-                    <Card fluid>
+                  <Link to='/technical/domains'>
+                    <Card fluid className='technical-card'>
+                      <img src='http://techcity.com/wp-content/uploads/2015/09/tech-support_tech-city-new-york-services2.jpg' />
                       <Card.Content>
                         <Card.Header>
-                          Knowledge Base
+                          Domains
                         </Card.Header>
                         <Card.Description>
-                          Description
+                          Browse courses by domain
+                        </Card.Description>
+                      </Card.Content>
+                    </Card>
+                  </Link>
+                </Grid.Column>
+
+                <Grid.Column>
+                  <Link to='/technical/languages'>
+                    <Card fluid className='technical-card'>
+                      <img src='https://az616578.vo.msecnd.net/files/2016/10/31/636135297934836257-753844635_cs.jpg' />
```

Comment:
Haha, let's try not to do this.  It is a good practice to find the pictures from some royalty free image distribution website like https://www.pexels.com/.